lib/kadira.js

```diff
@@ -63,6 +63,11 @@ Kadira.connect = function (appId, appSecret, options) {
   options.recordIPAddress = options.recordIPAddress || 'full';
   options.eventStackTrace = options.eventStackTrace || false;
   options.stalledTimeout = options.stalledTimeout || 1000 * 60 * 30;
+  options.disableClientErrorTracking = options.disableClientErrorTracking || false;
+
+  console.trace();
+
+  console.log(options);
```

Comment:
Yeap, still need to figure some things out though, forgot to set this to a draft PR, sorry.

Reply:
Does the overall solution look ok at this point?

Reply:
It does look good.

I'm not sure if we need `Kadira.disableClientErrorTracking` and `Kadira.enableClientErrorTracking`. Calling `Kadira.disableErrorTracking` on the client does mostly the same thing. The only difference is calling `Kadira.disableClientErrorTracking` on the server would ensure it is disabled immediately, instead of there being a small delay on the client where it could track an error before `Kadira.disableErrorTracking` is called.